communication/marketing-team/OWNERS

```diff
@@ -4,13 +4,13 @@ reviewers:
   - chris-short
   - onlydole
   - parispittman
-  - mbbroberg
 approvers:
   - parispittman
-  - mbbroberg
   - Pensu
   - rajula96reddy
   - kaslin
+emeritus_approvers:
+  - mbbroberg
```

Comment:
cc @mbbroberg Would you like to remain as an emeritus approver in the marketing team owners file?